src/tracks/playabletrack/wavetrack/ui/WaveTrackAffordanceControls.cpp

```diff
@@ -52,6 +57,32 @@ class WaveTrackAffordanceHandle final : public AffordanceHandle
     }
 };
 
+class SetWaveClipNameCommand : public AudacityCommand
```

Comment:
I see you made this command by analogy with SetTrackNameCommand in CommonTrackControls.cpp, and like that command class, it isn't meant to be registered with CommandManager.

It is commented there, that the same effect can be achieved with other commands available to macro programming.

Is that so for change of clip name?

No.  So I suggest: the existing SetClipCommand class needs to be extended with another parameter for setting the name, by means of macro programming.

I know that wasn't part of the specification of this project, but it is something that should be done for completeness: exposing all changes of properties we can to macro programmers too.

This could be a separat PR.